policyengine_us/reforms/congress/wyden_smith/ctc_expansion.py

```diff
@@ -0,0 +1,107 @@
+from policyengine_us.model_api import *
+
+
+def create_ctc_expansion() -> Reform:
+    class refundable_ctc(Variable):
+        value_type = float
+        entity = TaxUnit
+        label = "refundable CTC"
+        unit = USD
+        documentation = (
+            "The portion of the Child Tax Credit that is refundable."
+        )
+        definition_period = YEAR
+        reference = "https://www.law.cornell.edu/uscode/text/26/24#d"
+
+        def formula(tax_unit, period, parameters):
+            # This line corresponds to "the credit which would be allowed under this section [the CTC section]"
+            # without regard to this subsection [the refundability section] and the limitation under
+            # section 26(a) [the section that limits the amount of the non-refundable CTC to tax liability].
+            # This is the full CTC. This is then limited to the maximum refundable amount per child as per the
+            # TCJA provision.
+
+            ctc = parameters(period).gov.irs.credits.ctc
+
+            maximum_amount = tax_unit("ctc_refundable_maximum", period)
+
+            if ctc.refundable.fully_refundable:
+                reduction = tax_unit("ctc_phase_out", period)
+                return max_(0, maximum_amount - reduction)
+
+            total_ctc = tax_unit("ctc", period)
+            maximum_refundable_ctc = min_(maximum_amount, total_ctc)
+
+            # The other part of the "lesser of" statement is: "the amount by which [the non-refundable CTC]
+            # would increase if [tax liability] increased by tax_increase", where tax_increase is the greater of:
+            # - the phase-in amount
+            # - Social Security tax minus the EITC
+            # First, we find tax_increase:
+
+            earnings = tax_unit("tax_unit_earned_income", period)
+            earnings_over_threshold = max_(
+                0, earnings - ctc.refundable.phase_in.threshold
+            )
+            qualifying_children = tax_unit("ctc_qualifying_children", period)
+
+            relevant_rate = min_(ctc.refundable.phase_in.rate * qualifying_children, 100)
```

Comment:
The tweet didn't indicate it was capped at 100% phase-in rate. If it were, it would be capped at 1 not 100.
```suggestion
            phase_in_rate = ctc.refundable.phase_in.rate * qualifying_children
```